pandas/core/generic.py

```diff
@@ -133,7 +132,7 @@
 from pandas.core import (
     algorithms as algos,
     arraylike,
-    common as com,
+    common,
```

Comment:
does it work to just import the various functions from `pandas.core.common`? `from pandas.core.common import get_rename_function, ...`?